### DISCUSSION TITLE
可以加入 A-B 播放模式吗？

### DISCUSSION THREAD

**Discussion by conglinyizhi** [the1812/Bilibili-Evolved#2235] [功能建议 / Ideas]
就是在播放器达到指定位置（B）的时候就直接跳转到某个位置（A），我之前尝试做过那样的插件但是现在用那种方法会导致我的浏览器崩溃，感谢大佬提供的插件~

**Comment by CKylinMC** [comment]
[哔哩哔哩ab循环](https://greasyfork.org/scripts/422365)

自荐一下。

> AB循环，在某两个进度之间循环！进阶版洗脑循环！
> 
> 用法：
> 
> 在你想开始循环的地方暂停，然后点击菜单的“设置A点”或键盘数字1（不允许小键盘）（不设置默认从开头开始）
> 在你想结束的地方暂停，然后点击菜单的“设置B点”或键盘数字2（不允许小键盘）（不设置默认播放到视频结束）
> 点击菜单的“开始循环”。需要停止时只需要点击菜单的“停止循环”或键盘数字3（不允许小键盘）即可。
> 提示：
> 
> 脚本不会干涉用户的操作，开启循环后，依然可以暂停视频等操作。
> 未设置B点时，默认结束点为视频总时长-0.1s，防止触发B站自动暂停。
> 脚本会在选项中显示当前选择的秒数。
> 循环时依然可以设置A、B点，立刻生效。

**Reply by conglinyizhi** [reply]
感谢您的回复

目前已经在使用这个插件，按下键盘在 Firefox 上 *似乎* 没有作用，不过不影响使用，感谢您提供的插件！
有的视频我真的不喜欢看片头和片尾，A-B 循环的确在用的时候挺好用的，~~为啥官方不做，真的是……~~

**Reply by CKylinMC** [reply]
按键BUG我刚刚注意到了，我去修复一下
